framework/src/modules/token/schemas.ts

```diff
@@ -200,6 +208,12 @@ export const crossChainTransferParamsSchema = {
 			dataType: 'uint64',
 			fieldNumber: 6,
 		},
+		messageFeeTokenID: {
+			dataType: 'bytes',
+			fieldNumber: 7,
+			minLength: TOKEN_ID_LENGTH,
+			maxLength: TOKEN_ID_LENGTH,
```

Comment:
Just checking, is it necessary to have both `min` and `max` len equal to the same value or we can have a single value `len`?

Reply:
on JSON schema there is no `len` definition